### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by alexmaragko** [Kruemelkatze/vscode-dashboard#87]
It would be great to be able to fully navigate and select a project using only the keyboard.
For example, after activating the dashboard (ctrl+F1)  use tab, or arrow keys to navigate and select a project.

**Comment by Kruemelkatze** [comment]
I tried around a bit on how this would work. Unfortunately, with the line breaks done with flexbox, i can't easily determine which project tile is in the desired direction without having to iterate over each and checking their coordinates. Which feels really hacky tbh.

But if you can come up with a solution for this, I'll gladly have a look at it!

**Comment by ege-del** [comment]
How about putting each group's element into arrays and selecting with that?
e.g;
Up key, selects the last tile of group above
Down key, selects the first tile of group below
Left key, selects the tile with -1 index in array
Right key, selects the tile with +1 index in array

**Comment by Kruemelkatze** [comment]
Yeah, tried that at first, but feels really awkward if your group spans across multiple rows. The colleagues I tested this feature with instinctively used the down key to get to the next row, not the next group.
